examples/webgpu_compute_particles.html

```diff
@@ -32,6 +32,7 @@
 			import WebGPURenderer from 'three/addons/renderers/webgpu/WebGPURenderer.js';
 
 			import { OrbitControls } from 'three/addons/controls/OrbitControls.js';
+			import Status from 'three/addons/libs/stats.module.js';
```

Comment:
Please import the module as `Stats` not `Status`.

Reply:
> Please import the module as `Stats` not `Status`. Otherwise the import is not consistent compared to all other examples.

Sorry, my mistake...